core/src/main/java/org/frankframework/jdbc/JdbcQuerySenderBase.java

```diff
@@ -537,7 +533,7 @@ protected PipeRunResult getResult(ResultSet resultset, Object blobSessionVar, Ob
 				db2document.setGetBlobSmart(isBlobSmartGet());
 				db2document.writeDocument(getOutputFormat(), getDbmsSupport(), resultset, getMaxRows(), isIncludeFieldDefinition(), target, isPrettyPrint());
 			}
-			target.close();
+			target.close(); // Have to close early! B/c otherwise the result is not complete!
```

Comment:
yea, not complete is maybe a bit vague, but it calls closeDocument, which otherwise would be called after the message has been read/processed.

Reply:
Ik zal het comment bijwerken! Maakt het inderdaad duidelijker.